### ISSUE TITLE
VPC deployment failes

### ISSUE THREAD

**Issue by carloslme** [terraform-google-modules/terraform-google-network#496]
### TL;DR

Error when I run a command to create a deployment (VPC) using the [documentation](https://cloud.google.com/infrastructure-manager/docs/deploy-vpc-with-terraform#create_a_deployment)

### Expected behavior

No errors, only the message `Creating the deployment...done.` in console.

### Observed behavior

I got this:
`ERROR: (gcloud.alpha.infra-manager.deployments.apply) Revision failed: The apply build 676f35a6-ce03-48a2-8613-cb10c10a3903 failed while running: fetch.
blueprint fetch failed: error downloading 'https://github.com/terraform-google-modules/terraform-google-network.git?ref=main': /usr/bin/git exited with 1: error: pathspec 'main' did not match any file(s) known to git.`

### Terraform Configuration

```hcl
I executed this code on Cloud Shell in GCP:
`gcloud alpha infra-manager deployments apply projects/MY-PROJECT/locations/us-central1/deployments/quickstart-deployment \
    --service-account=projects/MY-PROJECT/serviceAccounts/infra-manager-test@MY-PROJECT.iam.gserviceaccount.com \
    --git-source-repo=https://github.com/terraform-google-modules/terraform-google-network \
    --git-source-directory=modules/vpc \
    --git-source-ref=main \
    --input-values=project_id=MY-PROJECT,network_name=quickstart-vpc`
```

### Terraform Version

```sh
Terraform v1.5.6
on linux_amd64

Your version of Terraform is out of date! The latest version
is 1.5.7. You can update by downloading from https://www.terraform.io/downloads.html
```

### Additional information

I guess the problem is that the branch `main` does not exist in the repo (`master` does exist).
Please update the documentation with:
`
I executed this code on Cloud Shell in GCP:
`gcloud alpha infra-manager deployments apply projects/MY-PROJECT/locations/us-central1/deployments/quickstart-deployment \
    --service-account=projects/MY-PROJECT/serviceAccounts/infra-manager-test@MY-PROJECT.iam.gserviceaccount.com \
    --git-source-repo=https://github.com/terraform-google-modules/terraform-google-network \
    --git-source-directory=modules/vpc \
    --git-source-ref=master \
    --input-values=project_id=MY-PROJECT,network_name=quickstart-vpc``
